### PR TITLE
csound: add HEAD

### PR DESCRIPTION
**Pull request by nwhetsell**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This adds ~Csound documentation, HRTF data files (**h**ead-**r**elated **t**ransfer **f**unction, for HRTF opcodes like [`hrtfstat`](https://csound.com/docs/manual/hrtfstat.html)), and~ a HEAD option.

### CODE REVIEW

File: Formula/csound.rb [hunk 1]
```diff
@@ -37,6 +38,21 @@ class Csound < Formula
     sha256 "2716e916a9dd9445b2a4de1f2325da818b7f097ec7004d453c83b10205167100"
   end
 
+  resource "csound-manual-html" do
```
Comment (fxcoudert) [inline Formula/csound.rb]: If the doc can be built & installed from the standard archive, we'll do that. If not, let's not add to the maintenance burden of the formula by adding additional resources (especially three versions of the doc).

File: Formula/csound.rb [hunk 2]
```diff
@@ -68,31 +84,45 @@ def install
...
     libexec.install "#{buildpath}/interfaces/ctcsound.py"
 
-    version = Language::Python.major_minor_version "python3"
-    (lib/"python#{version}/site-packages/homebrew-csound.pth").write <<~EOS
+    python_version = Language::Python.major_minor_version "python3"
+    (lib/"python#{python_version}/site-packages/homebrew-csound.pth").write <<~EOS
       import site; site.addsitedir('#{libexec}')
     EOS
+
+    (pkgshare/"hrtf-data").install Dir["samples/*.dat"]
```
Comment (fxcoudert) [inline Formula/csound.rb]: If this is suitable to install, it should be installed by `make install`.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/csound.rb (hunk 1 above)
Comment (nwhetsell) [reply Formula/csound.rb]: Alas, the Csound’s docs are in a separate repository (https://github.com/csound/manual). I’ve removed all this.

File: Formula/csound.rb (hunk 2 above)
Comment (nwhetsell) [reply Formula/csound.rb]: Understood, I’ll open an issue upstream.

**Comment by nwhetsell** [comment]
I’ve backed out the changes related to docs and HRTF files, which means this PR really only adds a HEAD option. (I think this means a revision bump isn’t necessary, so I backed that out too.)

Thanks for the very fast review and for all your work on Homebrew!

**Comment by fxcoudert** [comment]
Thanks @nwhetsell

**Pull request merged**